### PR TITLE
Rwanda case study

### PR DESCRIPTION
**Pull request by Norric1Admin**
@brockfanning Country case study for Rwanda

### CODE REVIEW

**Comment by brockfanning** [comment]
@Norric1Admin We should also add it to the [navigation](https://github.com/open-sdg/open-sdg/blob/master/mkdocs.yml#L48), and to the [community](https://github.com/open-sdg/open-sdg/blob/master/docs/community.md) page, right?

**Comment by Norric1Admin** [comment]
@brockfanning Ah yes, probably a wise move! Thanks.

File: docs/case-studies/rwanda.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+We have a strategic partnership with the UK Office for National Statistics (ONS) and they worked with the Centre for Open Data Enterprise (CODE) to set up the platform for us. The actual switch of our live site to Open SDG was done during a workshop with ONS in Rwanda.
+
+## How long did it take to set up the platform?
+Adopting the SDG open data did not take long. Just one day to set up the platform. But its updation is ongoing. 
+
+## How do you upload data and metadata into the platform?
+We have data from previous systems (eg DevInfo) as Excel files and receive data from MDAs in various formats. ONS ran a training session and helped us use Excel and scripts (Python and VBA) to convert the data and metadata into the required format and troubleshoot any errors during the upload process.
+
+## How is the platform hosted? (GitHub, local server, other platform)
+Currently the �staging� and public �production� versions are both hosted on GitHub.
```
Comment (brockfanning) [inline docs/case-studies/rwanda.md]: @Norric1Admin When I try to build locally it complains about these smart-quotes. To be safe could we change to regular quotes?

Comment (Norric1Admin) [reply docs/case-studies/rwanda.md]: Thanks - I didn't notice those. Amended and ready to go.

**Pull request merged**